practice1/pr1.html

```diff
@@ -37,30 +37,88 @@
         </div>
         <hr>
     <section class="blocks-layout">
-        <div class="block-caption">
-            <h3 class="secondary-header">Spectacles designed for faces. The average ones, the unaverage ones and the very unaverage ones too.</h3>
-            <a href="">Shop spectacles</a>
+        <div>
+            <h3 class="secondary-header">Spectacles<br>designed for faces.<br>The average ones, the unaverage ones and the very unaverage ones too.</h3>
+                <a href="" class="caption-small shop-btn-container"><img src="img/Frame (2).svg" alt="Shop here" class="shop-btn" />Shop spectacles</a>
         </div>
         <div class="block-bradley img-text">
-            <h4 class="caption-small">Narrow bridge</h4>
+            <div class="caption-favorite">
```

Comment:
You really like the word **caption**, don't you?
I would think towards something like `product-card-header`.
Plus, would probably be more semantically correct to change `div` to `header`